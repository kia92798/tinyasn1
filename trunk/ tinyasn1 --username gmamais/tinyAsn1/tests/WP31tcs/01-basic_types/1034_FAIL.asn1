--FSemantic Error: values must be distinct
--EMY-MODULE DEFINITIONS ::= BEGIN
--E&nbsp;&nbsp;Int ::= INTEGER {
--E&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;value1(0),
--E&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;value2(2),&nbsp;&nbsp;
--E&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;value3(2)&nbsp;&nbsp;
--E&nbsp;&nbsp;&nbsp;&nbsp;}
--EEND

MY-MODULE DEFINITIONS ::= BEGIN

	Int ::= INTEGER {
			value1(0),
			value2(2),	
			value3(2)	-- values must be distinct 
		}
END
