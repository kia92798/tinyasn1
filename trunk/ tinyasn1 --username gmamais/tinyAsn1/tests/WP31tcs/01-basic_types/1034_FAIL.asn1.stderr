Error in line:12 named number items: value2 and value3 have the same value(2)
